public/app/features/explore/TraceView/components/model/link-patterns.tsx

```diff
@@ -124,7 +124,7 @@ function callTemplate(template: ProcessedTemplate, data: any) {
 export function computeTraceLink(linkPatterns: ProcessedLinkPattern[], trace: Trace) {
   const result: TLinksRV = [];
   const validKeys = (Object.keys(trace) as Array<keyof Trace>).filter(
-    (key) => typeof trace[key] === 'string' || trace[key] === 'number'
+    (key) => typeof trace[key] === 'string' || typeof trace[key] === 'number'
```

Comment:
@aocenas looks like you were the original author here... can you confirm if this was the original intention? 🤔 i think it's the only thing that makes sense 😅

Reply:
Yup I think that was a bug. Checking if value of some property on `trace` object is literally `"number"` string does not make much sense here and it also validates the values if they can be interpolated (which is true for simple value like strings and numbers) so I think this change makes sense.